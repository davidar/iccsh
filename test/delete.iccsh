:declare x int x
:eval x = 42
:info x
:printf %d x

:delete x
:declare x const char *x
:eval x = "hello"
:info x
:printf %s x
