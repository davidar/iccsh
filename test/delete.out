int x
42
const char *x
hello
